runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -241,9 +241,15 @@ local function on_line_impl(self, buf, line, is_spell_nav)
     end
 
     while line >= state.next_row do
-      local capture, node, metadata = state.iter()
+      local capture, node, metadata = state.iter(line)
 
       if capture == nil then
+        if node then
+          local start_row = node:range()
```

Comment:
This should use the `get_range()` from below. I know `metadata` will be `nil`.

I might be better to refactor by removing the break so the logic isn't duplicated.

Reply:
Right, I forgot about Lua's amazing tolerance for nulls